ta-sdk-spi/src/main/java/com/ibm/ta/sdk/spi/collect/AssessmentUnitMetadataJson.java

```diff
@@ -0,0 +1,64 @@
+package com.ibm.ta.sdk.spi.collect;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonElement;
+import com.google.gson.annotations.Expose;
+
+import java.util.Properties;
+
+/**
+ * Assessment unit metadata file required by Transformation Advisor.
+ */
+public class AssessmentUnitMetadataJson {
+
+  @Expose
+  private String domain;
+
+  @Expose
+  private String middleware;
+
+  @Expose
+  private String dcVersion = "2.1.3";
```

Comment:
How come the dcVersion is being hardcoded?

Reply:
Updated hardcode value to 2.0.4. Will change in the future on how we get this (read file or env variable)